src/components/certify/Certify.vue

```diff
@@ -191,6 +191,16 @@ export default class Certify extends Vue {
 #AR-step-4-container {
   line-height: 1.2rem;
   font-size: $px-16;
+
+  &.invalid-section {
+    border-left: 3px solid $app-red !important;
+    border-top-left-radius: 0 !important;
+    border-bottom-left-radius: 0 !important;
+  }
+
+  .error-text {
+    color: $app-red;
+  }
```

Comment:
Aren't both of these classes already present in the base SCSS, both here in bcrs-shared-components and in the UI projects where this component is imported?

Reply:
Yes, they are. I just wanted to make sure they do. Give me a minute. Let me check if they are persisted on storybook. I'll remove it if they are.

Reply:
![image](https://user-images.githubusercontent.com/51458187/185718303-7a05b75e-3925-41db-b235-9cb031fc5f54.png)

I removed it and checked storybook. Somehow they weren't there anymore. That's why it isn't shown on Dev either.

Dev below:

![image](https://user-images.githubusercontent.com/51458187/185718358-dd14d369-b250-4633-99e2-c6e13b43977f.png)

Reply:
I think they were not persisted in this component, so this code just fixed it.

Reply:
Dev and Storybook aren't the same thing. Storybook is a project in THIS repo that loads this component. Dev could be Filings/Create/Edit/other UI that uses this component.

Please add these classes to the bcrs-shared-components assets.

Also, if you identify a UI that doesn't display these classes, add the classes to that project.

We don't want the duplicate class declarations in the component and base assets.

Reply:
> I think they were not persisted in this component, so this code just fixed it.

I'm not sure what you mean by "persisted". This component should use the base classes when it's imported in Storybook or any other UI.

Reply:
This component isn't displaying the error bar and texts.

Reply:
The errors classes should be global, in this project (for Storybook to display your component) and in any UI project that uses this shared component.

I do not want to see local declarations of the error classes.

Reply:
Definitely agree. 
This class will be overridden by the local definition either way, so it makes sense to add this class to the storybook assets styles (if it doesn't already exist), since this class will only be used for the storybook instance afaik.

Reply:
@cameron-freshworks The classes "invalid-section" and "error-text" are available in the base.scss file: [here](https://github.com/bcgov/bcrs-shared-components/blob/5c1623427f435c7b730d04c79aa06e82344a24ff/src/assets/styles/base.scss#L286). In the original implementation, the style "error-text" didn't take place (turn the text into red) when the variable "invalidSection" is false.

Reply:
🤔 
To that point, isn't that expected. Shouldn't the class binding only take effect when the variable is `true`?

Reply:
I am guessing the reason is because of"error-text" is added to to "title-label" for the "Legal Name" and "certify-stmt" for the comments "I, [Legal Name] ...". The local class becomes ".title-label.error-text" and overrides "error-text" from the globally defined style.

Reply:
This is the reason why I created this ticket on Friday and wanted to give it a quick fix. Create UI is using this component, and the error bar and texts are not turning red when it sends "invalidSection = true" to this component.

Reply:
Interesting. 

Would this be a scenario where the `color` could be defined with `!important` locally. 
Or potentially this `error-text` defined in the `overrides`?

Reply:
This second image shows the result using the original code as below (I removed the coded local css style):

![image](https://user-images.githubusercontent.com/51458187/185981099-67e98ff3-b9a6-40fa-b2d1-3563cc3836a0.png)